.github/workflows/ocserv-version-update.yml

```
@@ -0,0 +1,33 @@
+name: bump ocserv version
+on:
+  schedule:
+  - cron: "0 0 * * *"
+jobs:
+  ocserv-version-update:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: write
+      pull-requests: write
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Update openconnect version
+      id: get_version
+      run: |
+        OC_VERSION=$(curl https://gitlab.com/api/v4/projects/openconnect%2Focserv/repository/tags | jq -r '.[0].name')
```

Let's change this to the same url we use in Dockerfile.
```suggestion
        OC_VERSION=$(curl -s ftp://ftp.infradead.org/pub/ocserv/ | awk '{print $9}' | tail -n 1 | sed 's:.*-::;s:.tar.*::')
```